release-notes/9.0/supported-os.json

```diff
@@ -0,0 +1,304 @@
+{
+    "channel-version": "9.0",
+    "last-updated": "2024-05-01",
+    "operating-systems": [
+        {
+            "family": "Android",
+            "distributions": [
+                {
+                    "name": "Android",
+                    "lifecycle-policy": "https://support.google.com/android",
+                    "architectures": [
+                        "x64",
+                        "arm64",
+                        "arm"
+                    ],
+                    "supported-cycles": [
+                        "API 21+"
+                    ]
+                }
+            ]
+        },
+        {
+            "family": "Apple",
+            "distributions": [
+                {
+                    "name": "iOS",
+                    "lifecycle-policy": "https://support.apple.com/ios/",
+                    "architectures": [
+                        "arm64"
+                    ],
+                    "supported-cycles": [
+                        "12.2"
```

Comment:
I would expect here the versions of 15+. 12.2 through 14.5 are already EOL. Maybe this is also something like a min-sdk version? I haven't used iOS builds myself so far. At least that seams to correspond to the `SupportedOSPlatformVersion`.

Reply:
No, we still support iOS 12.2 as the minimum target an app can run on. It is similar to min-sdk on Android.